pkg/config/config.go

```diff
@@ -1207,3 +1223,13 @@ func (c *Config) FindInitBinary() (string, error) {
 	}
 	return c.FindHelperBinary(defaultInitName, true)
 }
+
+// PodmanshTimeout returns the timeout in seconds for podmansh to connect to the container.
+// Returns podmansh.Timeout if set, otherwise engine.PodmanshTimeout for backwards compatability.
```

Comment:
typo causing codespell errors.

Reply:
Once touched up @grisu48, the tests should pass, and we can merge this.

Reply:
`/s/compatability/compatability`. Gosh, if I would get a nickel every time I make this mistake :man_facepalming: :wink: 

Fixed.